src/RE/B/BSVisit.cpp

```diff
@@ -65,6 +65,7 @@ namespace RE
 			return result;
 		}
 
+#ifndef SKYRIM_CROSS_VR
```

Comment:
The proper fix for this function is to change `node->children` to `node->GetChildren()`

I did that in this PR https://github.com/alandtse/CommonLibVR/pull/63